spec/facility_spec.rb

```diff
@@ -23,4 +32,76 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  #facility_1 & facility_2
+  it 'can register vehicles' do
+    expect(facility_1.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+    expect(cruz.registration_date).to eq(nil)
+    expect(facility_1.registered_vehicles).to eq([])
+    expect(facility_1.collected_fees).to eq(0)
+    facility_1.register_vehicle(cruz)
+    expect(cruz.registration_date).to eq(Date.today.year)
+    expect(cruz.plate_type).to eq(:regular)
+    expect(facility_1.registered_vehicles).to eq([cruz])
+    expect(facility_1.collected_fees).to eq(100)
+    facility_1.register_vehicle(camaro)
```

Comment:
Is there a way that you could break this big test into smaller tests? Like a test for `collected fees`, `adding services`, and `registering a vehicle`?? Great job being robust though.